pkgs/top-level/all-packages.nix

```diff
@@ -21467,6 +21475,10 @@ with pkgs;
     stdenv = clangStdenv;
   };
 
+  vte-gtk4 = vte.override {
+    gtk3 = gtk4;
```

Comment:
Thats really confusing

Reply:
I switched to passing both and using an extra attribute for control, hopefully, that is better.